assignments/lambda-classes.js

```diff
@@ -1 +1,175 @@
 // CODE here for your Lambda Classes
+
+
+//person class original
+
+class Person{
+    constructor(persFact){
+        this.name = persFact.name;
+        this.age = persFact.age;
+        this.location = persFact.location;
+    }   
+
+    speak(){
+        return  `Hello my name is ${this.name}, I am from ${this.location}`
+    }
+}
+
+
+//instructor class  extend-> of person
+
+class Instructor extends Person{
+    constructor(instFacts){
+        super(instFacts);
+        this.specialty = instFacts.specialty;
+        this.favLanguage = instFacts.favLanguage;
+        this.catchPhrase = instFacts.catchPhrase;
+    }
+
+    demo(subject){
+        return `Today we are learning about ${subject}`
+    }
+
+    grade(name, subject){
+        return `${name} receives a perfect score on ${subject}`
+    }
+
+    randomegrade(student){
+        return Math.ceil(Math.random(student.grade) * 100);
+    }
+
+    meangrade(student, instructor){
+        return `${student.name}'s old grade was ${student.grade}. ${student.name}'s new grade is ${instructor.randomegrade(Student)}`
+    }
+}
+
+
+//student class extend-> of person
+class Student extends Person{
```

Comment:
nice work using the Class constructor. In the future we will be passing properties from one component to another and it would be a good idea getting used to using the term "props". It is a universal term for properties